L1Trigger/Phase2L1ParticleFlow/src/PFAlgo2HGC.cc

```diff
@@ -193,6 +193,8 @@ void PFAlgo2HGC::link_tk2mu(Region &r, std::vector<int> &tk2mu, std::vector<int>
     int imatch = -1;
     for (int itk = 0, ntk = r.track.size(); itk < ntk; ++itk) {
       const auto &tk = r.track[itk];
+      if (!tk.quality(l1tpf_impl::InputTrack::PFLOOSE))
```

Comment:
Same comment about "configurability" here.

Reply:
Sorry I have missed the original comment probably. I meant that these minimal quality cuts (`PFLOOSE `) could be made configurable if needed (there's another spot where this is true, I need to re-find it).

Reply:
the way things are now, people can configure in the upstream code what gets written in these quality flags.
then there's no real need of re-configuring what quality flags are used here downstream.

Reply:
I see thanks